.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,4 +2,6 @@
 
 * [ ] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
 * [ ] The changes are covered by a new or existing set of unit tests which follow the Arrange-Act-Assert syntax such as is used [in this example](https://github.com/fluentassertions/fluentassertions/blob/daaf35b9b59b622c96d0c034e8972a020b2bee55/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33).
-* [ ] If the contribution affects [the documentation](https://github.com/fluentassertions/fluentassertions/tree/master/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com).
+* [ ] If the contribution adds a feature or fixes a bug, please update [**the release notes**](https://github.com/fluentassertions/fluentassertions/tree/master/docs), which are published on the [website](https://fluentassertions.com/releases).
```

Comment:
@eNeRGy164 Did you miss this line about release notes?

Reply:
Check, sorry!

Reply:
No worries, I'm grateful for the extra pair eyes